### PR TITLE
Crud jira255

### PR DESCRIPTION
**Pull request by prem43**

### CODE REVIEW

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupCategoryService.cs [hunk 1]
```diff
@@ -15,10 +15,10 @@ public interface ILookupCategoryService
         /// <returns></returns>
         List<LookupCategoryModel> Get();
         /// <summary>
-        /// Get Record By Id From LookupCategory
+        /// Get All Records From LookupCategory
         /// </summary>
-        /// <param name="id"></param>
         /// <returns></returns>
+
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupCategoryService.cs:21]: why there is wrong comment and extra whitespace

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupService.cs [hunk 2]
```diff
@@ -9,10 +10,35 @@ namespace TestingAndCalibrationLabs.Business.Core.Interfaces
     public interface ILookupService
     {
         /// <summary>
-        /// To Get All Records Of Lookup
+        /// Get All Record From Module
         /// </summary>
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupService.cs:14]: why you have putted wrong comment

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupService.cs [hunk 3]
```diff
@@ -9,10 +10,35 @@ namespace TestingAndCalibrationLabs.Business.Core.Interfaces
...
+        /// <param name="lookupModel"></param>
+        /// <returns></returns>
+        RequestResult<int> Create(LookupModel lookupModel);
+        /// <summary>
+        /// Delete Record In Module
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns></returns>
+        bool Delete(int id);
+
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupService.cs:40]: everywhere you have writted Wrong comment

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Common/LookupModel.cs [hunk 4]
```diff
@@ -13,9 +13,13 @@ public class LookupModel : Entity
...
-        /// It Contains The Id Of The Lookup Category
+        /// It Contains The Id Of The Lookup
         /// </summary>
         public int LookupCategoryId { get; set; }
-        
+        /// <summary>
+        /// It Contains Name Of Lookup
+        /// </summary>
+        public string LookupCategoryName { get; set; }
+
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Common/LookupModel.cs:23]: put proper comment man

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Data/Repository/Interfaces/BackOffice/ILookupRepository.cs [hunk 5]
```diff
@@ -10,6 +10,36 @@ public interface ILookupRepository
...
+        /// <returns></returns>
+        int Update(LookupModel lookupModel);
+        /// <summary>
+        /// Delete Record From Module By Id
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns></returns>
+        bool Delete(int id);
+
+
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Data/Repository/Interfaces/BackOffice/ILookupRepository.cs:43]: why you have putted wrong comment

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Tests/bin/Debug/netcoreapp3.1/appsettings.json [hunk 6]
```diff
@@ -9,7 +9,7 @@
   },
 
   "ConnectionStrings": {
-    "IdentityDBLocal": "Server=localhost;Database=GoogleDriveServiceDb; User ID=sa;Password=admin;"
+    "IdentityDBLocal": "Server=LENOVO;Database=jira-255 CRUD; User ID=sa;Password=admin;"
   },
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Tests/bin/Debug/netcoreapp3.1/appsettings.json:13]: dont push Connection string

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Common/Components/Grid/_gridTemplate2.cshtml [hunk 7]
```diff
@@ -46,7 +46,7 @@
                         }
                         <td>
                             <a class="fas fa-edit" style="margin:5px;color:rebeccapurple;" color:black;" asp-controller="Common" asp-action="Edit" asp-route-id="@item.Key"></a>|
-                            <a class="fa fa-trash" style="margin:5px;color:red;" data-toggle="modal" data-target="#deleteModalPopup" onclick="setValue(@item.Key)"></a>
+                            <a class="fa fa-trash" style="margin:5px;color:red;" @*data-toggle="modal" data-target="#deleteModalPopup"*@ onclick="setValue(@item.Key)"></a>
                         </td>
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Common/Components/Grid/_gridTemplate2.cshtml:50]: why there is commented code

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/LookupCategory/Delete.cshtml [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+        <dd class="col-sm-10">
+            @Html.DisplayFor(model => model.Name)
+        </dd>
+    </dl>
+
+    <form asp-action="Delete">
+        <input type="submit" value="Delete" class="btn btn-danger" /> |
+        <a asp-action="Index">Back to List</a>
+    </form>
+</div>
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/LookupCategory/Delete.cshtml:28]: why delete page is their I think you have to use modal popup

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupCategoryService.cs (hunk 1 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupCategoryService.cs:21]: i have fixed it.

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupService.cs (hunk 2 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupService.cs:14]: i have fixed it.

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupService.cs (hunk 3 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Interfaces/BackOffice/ILookupService.cs:40]: i have fixed it

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Common/LookupModel.cs (hunk 4 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Core/Model/Common/LookupModel.cs:23]: i have fixed it

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Data/Repository/Interfaces/BackOffice/ILookupRepository.cs (hunk 5 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Data/Repository/Interfaces/BackOffice/ILookupRepository.cs:43]: i have fixed it

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Tests/bin/Debug/netcoreapp3.1/appsettings.json (hunk 6 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Tests/bin/Debug/netcoreapp3.1/appsettings.json:13]: ok

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Common/Components/Grid/_gridTemplate2.cshtml (hunk 7 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Common/Components/Grid/_gridTemplate2.cshtml:50]: i hava fixed it

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/LookupCategory/Delete.cshtml (hunk 8 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/LookupCategory/Delete.cshtml:28]: The delete page is written in delete.cshtml like this

**Pull request merged**